src/main/java/de/tum/in/www1/artemis/web/rest/UserResource.java

```diff
@@ -174,18 +174,6 @@ public ResponseEntity<Void> setIrisAcceptedToTimestamp() {
         return ResponseEntity.ok().build();
     }
 
-    /**
-     * GET users/accept-iris : gets the irisAccepted flag for the user
-     *
-     * @return the ResponseEntity with status 200 (OK) and with body the flag's value as ZonedDateTime, or with status 404 (Not Found)
-     */
-    @GetMapping("users/accept-iris")
-    @EnforceAtLeastStudent
-    public ResponseEntity<ZonedDateTime> getIrisAcceptedForStudent() {
```

Comment:
Why do you no longer need this? Isn't this relevant for checking on the client if the user has already accepted the policy?

Reply:
Because it is sent to the client with the auth user object now. I eliminated this extra rest call